drivers/ad9144/ad9144.c

```diff
@@ -272,6 +272,95 @@ int32_t ad9144_setup_jesd204_link(struct ad9144_dev *dev,
 	return 0;
 }
 
+/*
+ * PLL fixed register writes according to table 17 of the
+ * AD9144 datasheet Rev. B.
+ */
+static const struct ad9144_reg_seq ad9144_pll_fixed_writes[] = {
+	{ 0x87, 0x62 },
+	{ 0x88, 0xc0 },
+	{ 0x89, 0x0e },
+	{ 0x8a, 0x12 },
+	{ 0x8d, 0x7b },
+	{ 0x1b0, 0x00 },
+	{ 0x1b9, 0x24 },
+	{ 0x1bc, 0x0d },
+	{ 0x1be, 0x02 },
+	{ 0x1bf, 0x8e },
+	{ 0x1c0, 0x2a },
+	{ 0x1c1, 0x2a },
+	{ 0x1c4, 0x7e },
+};
+
+static int ad9144_pll_setup(struct ad9144_dev *dev,
+			    const struct ad9144_init_param *init_param)
+{
+	unsigned int fref, fdac;
```

Comment:
Using fixed width integer types will help reduce code ambiguity when using different compilers and platforms. So we should use 'uint32_t' instead of 'unsigned int' and 'int32_t' instead of 'int'.